frontend/src/components/common/PostingDetails.tsx

```diff
@@ -61,20 +62,15 @@ const PostingDetails = ({
         </Text>
         <Text textStyle="body-regular">Point(s) of contact:</Text>
         <HStack pb={4}>
-          <PocCard
-            name="John Doe"
-            title="hard code"
-            email="hard code"
-            phoneNumber="hard code"
-            key={1}
-          />
-          <PocCard
-            name="John Doe"
-            title="hard code"
-            email="hard code"
-            phoneNumber="hard code"
-            key={1}
-          />
+          {postingDetails.employees?.map((employee) => (
```

Comment:
This should be safe, `employees` is not nullable.

```suggestion
          {postingDetails.employees.map((employee) => (
```